### PR TITLE
Add run-e2e-job script

### PR DESCRIPTION
**Pull request by kevinrizza**
-Add script to configure and run e2e tests for ci
-Add e2e-ci-job target to Makefile

### CODE REVIEW

File: scripts/run-e2e-job.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+MARKETPLACE_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+# Get operator-sdk binary.
+wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.2.0/operator-sdk-v0.2.0-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
+wget -O /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.11.3/bin/linux/amd64/kubectl && chmod +x /tmp/kubectl
```
Comment (tkashem) [inline scripts/run-e2e-job.sh]: Suggestion: Version number of k8s and sdk can be fed into the URLs from variables defined in the script.

**Comment by kevinrizza** [comment]
@aravindhp @tkashem I updated the script to use version variables for the binary downloads

File: scripts/run-e2e-job.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+set -o nounset
+set -o pipefail
+
+MARKETPLACE_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/..
+SDK_VERSION=v0.2.0
+KUBE_VERSION=v1.11.3
+
+# Get operator-sdk binary.
+wget -O /tmp/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VERSION}/operator-sdk-${SDK_VERSION}-x86_64-linux-gnu && chmod +x /tmp/operator-sdk
+wget -O /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl && chmod +x /tmp/kubectl
```
Comment (aravindhp) [inline scripts/run-e2e-job.sh]: Is `/tmp/` part of the `PATH` in the pod where the tests are running?

Comment (kevinrizza) [reply scripts/run-e2e-job.sh]: Yeah, putting these binaries in /tmp/ makes them available on the PATH in the ci container

**Comment by kevinrizza** [comment]
@tkashem could you review and approve when you have some time?

**Pull request merged**